### ISSUE TITLE
Suggestion - Books page should have filters or pagination.

### ISSUE THREAD

**Issue by bhaskarmac** [smashingmagazine/redesign#328]
**- Do you want to request a *feature* or report a *bug*?**
Feature
**- What is the current behaviour?**
Books page have lengthy scroll to page (https://next.smashingmagazine.com/ebooks/)

**- If the current behavior is a bug, please provide the steps to reproduce.**

**- What is the expected behaviour?**
Books page should have filters or the pagination.

**- Please mention your operating system version and the version of your browser**

**Comment by vitalyfriedman** [comment]
Good call! Adding it to our to-do list to add after all critical problems are solved. @bhaskarmac
